resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/interceptors/MessageSanitizerContainerResponseFilter.java

```diff
@@ -74,5 +76,18 @@ private String escapeXml(String str) {
         return sb.toString();
     }
 
+    private boolean containsHtmlText(ArrayList<Object> list) {
+       for (Iterator<Object> it = list.iterator(); it.hasNext();  ) {
+          Object o = it.next();
+          if (o instanceof String) {
+             String mediaType = (String) o;
+             String[] parts = mediaType.split("/");
+             if (parts.length >= 2 && parts[0].trim().equals("text") && parts[1].trim().startsWith("html")) {
```

Comment:
Just a question:
 * I haven't try it myself, but what if the content-type will be "TEXT/HTML", then containsHtmlText will return false and then the string in the entity (L36) will not be escaped? Other lines in this PR, who test content-type, use equalsIgnoreCase() method.